### PR TITLE
feat: integrate docsearch with algolia

### PR DESCRIPTION
**Pull request by rajkharvar**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -38,83 +38,61 @@ const config = {
...
-      },
-    },
-  ],],
-
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      algolia: {
+        appId: "JZUSBYXBFF",
+        apiKey: "0170bec8ff2a904d2ff209bbacab9f96",
```
Comment (yohanelly95) [inline docusaurus.config.js:46]: can you make this private @rajkharvar

**Review by yohanelly95 — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (rajkharvar) [reply docusaurus.config.js:46]: As per algolia, it's fine we can keep the api key public
https://support.algolia.com/hc/en-us/articles/8150525164049-Can-I-set-up-DocSearch-Github-secrets
https://support.algolia.com/hc/en-us/articles/18966776061329-Can-the-search-API-key-be-public

**Review by yohanelly95 — APPROVED** [review]

File: docusaurus.config.js [hunk 2]
```diff
@@ -38,83 +38,61 @@ const config = {
...
+        indexName: "razor",
+        contextualSearch: true,
+        // externalUrlRegex: 'external\\.com|domain\\.com',
```
Comment (ashish10677) [inline docusaurus.config.js:49]: Can you remove these if not required

File: docusaurus.config.js [hunk 3]
```diff
@@ -38,83 +38,61 @@ const config = {
...
-        //     to: "/docs/testnet/intro",
-        //     label: "Testnet",
-        //     position: "left",
-        //   },
-        //   {
-        //     to: "/docs/consume-data-feeds/introduction",
-        //     label: "Consume Data Feeds",
-        //     position: "left",
-        //   },
+          //   {
```
Comment (ashish10677) [inline docusaurus.config.js:65]: Why are these commented? If not required remove.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 2 above)
Comment (rajkharvar) [reply docusaurus.config.js]: Fixed in [e8499ad](https://github.com/razor-network/docs-v2/pull/132/commits/e8499addc3585d017da10b0de1d9d18d6a00eef6)

File: docusaurus.config.js (hunk 3 above)
Comment (rajkharvar) [reply docusaurus.config.js]: Removed [e8499ad](https://github.com/razor-network/docs-v2/pull/132/commits/e8499addc3585d017da10b0de1d9d18d6a00eef6)

**Review by ashish10677 — APPROVED** [review]

**Pull request merged**